src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -198,6 +200,17 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, policyMembers, r
         });
     }
 
+    if (policy?.areAccountingEnabled && policy?.areConnectionsEnabled && canUseAccountingIntegrations) {
```

Comment:
I think you might have forgotten to remove `areAccountingEnabled` that you might have used to use accounting feature switching?

`areAccountingEnabled` is the property that you can check to see if the accounting/connections feature is enabled or not.